prerequisites.rst

```diff
@@ -1,35 +1,280 @@
+======  =================  ==============  ==============  =======================================
+ OS      Debian Linux       oldoldstable    stable          Current stable (plus 1 month released)
+ BOOT    BIOS,UBOOT,UEFI    BIOS,mbr        BIOS,gpt        Disable Secure boot, gpt table
+ SDS     HDD,SSD,USB...     1, 4GiB        2, 120+500GiB    Disk drive with 120G root size, 8G swap size
+ DDS     HDD,SSD,USB...     0 or any        1 per share     One disk or part per shared resource
+ NET     LAN,WAN,SAN,VPN    LAN             SAN,PAN,LAN     Fiber IPv4, or at least cable LAN
+======  =================  ==============  ==============  =======================================
+
+The operating system (OS)
+^^^^^^^^^^^^^^^^^^^^^^^^^
+
+The |omv| is a piece of software, it resides and runs under a key software OS
+named Debian GNU/Linux as "the universal operating system" [9]_ 
+
+**Supported Debian OS versions can be checked in the** :doc:`releases section </releases>`.
+
+While possible to deploy in a virtual environment, depending of the nature of
+virtualization it will degrade the performance access in various ways. Installation
+in LXC or any other container based solution is not supported.
+
+Whatever the situation, the |omv| assumes that target operating system does not
+have any previous installation of any of the programs used by |omv| dependencies:
+
+=============  ==============  =================  ==========================================
+ software       package          related to        Observations
+=============  ==============  =================  ==========================================
+ http server    nginx           |webui|            port 80 must be free at installation
+ http system    php             |webui|            only OS php packages wil be used!
+ display GUI    lighdm,xdm..    Desktop install    **Conflicts, any will be deinstalled**
+ network man    netplan.io      Networking         network is managed by |omv| with netplan
+ ssh server     ssh             Remote access      any configuration will be managed
+ smb server     samba           Shared drives      any configuration will be managed
+ quota man      quota           Quota manage       any configuration will be managed
+=============  ==============  =================  ==========================================
+
+The device boot (BOOT)
+^^^^^^^^^^^^^^^^^^^^^^^^^
+
+The boot type and its support is determined by the Debian operating system
+which supports BIOS, UEFI, PXE, UBOOT and many others, the configurations
+of these depend on the installation of the operating system and once done
+the |omv| can be installed without problems.
+
+However the downloadable and ready-to-use ISO image from the official |omv| website
+only supports AMD64 with BIOS mode boot at the moment. If you want to have |omv| on
+other computers with another boot type/mode you should install Debian first
```

Comment:
```suggestion
other computers with another boot type/mode, then you should install Debian first
```

Reply:
commit this with 70f443af661407cfca3c794da2723e26d987bf2e